src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
+  } else if (days >= 3) {
+    result = result - 20;
+  }
```

Comment:
You can simplify the conditional logic by removing the `else` keyword. Since you are returning inside the `if` block, the subsequent code will not execute if the condition is met, making the `else` unnecessary.